src/app/app.component.html

```diff
@@ -1,3 +1,33 @@
 <ion-app>
-      <ion-router-outlet id="main-content"></ion-router-outlet>
+    <ion-menu side="start" contentId="main-content" type="overlay">
```

Comment:
Create a new module as layout and place all the headers , side menu and footer component.
Create new component for side menu and use that component here
Create new component for Header and footer and that component here